.github/workflows/container-publish.yaml

```diff
@@ -0,0 +1,105 @@
+name: Container image build and publish
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ "master" ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
```

Comment:
Why is needed to trigger on pull requests when it's skipped on line 25?

Reply:
Having this here allows for this Action to show up as `Skipped` in a PR, if its not listed then it shows up as `Failed`.

Reply:
Wouldn't it be displayed at all? For example we have this action - https://github.com/RedHatInsights/vulnerability-engine/blob/master/.github/workflows/semantic-release.yml, which is only on push and it's not showing in PRs at all.